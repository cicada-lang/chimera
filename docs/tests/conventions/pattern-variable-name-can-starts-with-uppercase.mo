print find [X, Y] {
  X = Y
}
