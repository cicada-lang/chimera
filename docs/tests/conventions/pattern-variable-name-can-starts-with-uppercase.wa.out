[[_.0, _.0]]
